src/base/static/components/molecules/place-list-item.js

```diff
@@ -0,0 +1,269 @@
+import React from "react";
+import styled from "react-emotion";
+import PropTypes from "prop-types";
+import { Button, IconButton } from "../atoms/buttons";
+import { HeartIcon } from "../atoms/icons";
+import { SmallTitle } from "../atoms/typography";
+import { UserAvatar } from "../atoms/imagery";
+import { RegularText, SmallText, Link } from "../atoms/typography";
+import { placeConfigSelector } from "../../state/ducks/place-config";
+import { supportConfigSelector } from "../../state/ducks/support-config";
+import { appConfigSelector } from "../../state/ducks/app-config";
+import { connect } from "react-redux";
+import { translate } from "react-i18next";
+import { HorizontalRule } from "../atoms/layout";
+import sharePlace from "../../utils/share-place";
+
+const PlaceContainer = styled("div")({
+  display: "flex",
+  flexDirection: "column",
+  overflow: "hidden",
+  margin: "0px 16px 16px 16px",
```

Comment:
On my machine in Chrome I think the very top and bottom of the `PlaceContainer` components are getting sliced off:

<img width="615" alt="screen shot 2018-11-05 at 3 31 44 pm" src="https://user-images.githubusercontent.com/9222338/48033397-0dad7d00-e110-11e8-807e-bf4325662c89.png">

Maybe we can add a `paddingTop` and `paddingBottom` of `10px` or so to prevent that?

Reply:
Good eye! Yes, I'll clean that up